src/predefinedComponents/TabbedHeader/TabbedHeader.js

```diff
@@ -157,12 +158,22 @@ export default class TabbedHeader extends React.Component {
       tabWrapperStyle,
       tabsContainerStyle,
       onRef,
+      keyboardShouldPersistTaps,
+      scrollRef,
+      contentContainerStyles,
+      refreshControl,
+      rememberTabScrollPosition,
     } = this.props;
 
     return (
       <>
         <StatusBar barStyle="light-content" backgroundColor={backgroundColor} translucent />
         <StickyParallaxHeader
+          rememberTabScrollPosition={rememberTabScrollPosition}
```

Comment:
This prop looks strange. Are you sure that this one is valid? 🤔

Reply:
Seems to already exist on `TabbedHeaderProps` as `rememberTabScrollPosition?: boolean;`